southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImpl.java

```diff
@@ -8,30 +8,150 @@
 
 package org.opendaylight.ansible.southbound;
 
+import static org.opendaylight.ansible.mdsalutils.Datastore.OPERATIONAL;
+
+import ch.vorburger.exec.ManagedProcess;
+import ch.vorburger.exec.ManagedProcessBuilder;
+import ch.vorburger.exec.ManagedProcessException;
 import com.google.common.util.concurrent.ListenableFuture;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.AnsibleCommandService;
-import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandInput;
-import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandOutput;
-import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandOutputBuilder;
+import org.opendaylight.ansible.mdsalutils.RetryingManagedNewTransactionRunner;
+import org.opendaylight.controller.md.sal.binding.api.DataBroker;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.yang.types.rev130715.Uuid;
+import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.*;
+import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.commands.Command;
+import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.commands.CommandBuilder;
+import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.commands.CommandKey;
+import org.opendaylight.yangtools.yang.binding.InstanceIdentifier;
 import org.opendaylight.yangtools.yang.common.RpcResult;
 import org.opendaylight.yangtools.yang.common.RpcResultBuilder;
+import org.ops4j.pax.cdi.api.OsgiService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @Singleton
 public class AnsibleCommandServiceImpl implements AnsibleCommandService {
     private static final Logger LOG = LoggerFactory.getLogger(AnsibleCommandServiceImpl.class);
+    public static final InstanceIdentifier<Command> ANSIBLE_COMMAND_PATH = InstanceIdentifier.create(Commands.class)
+            .child(Command.class);
+    private Map<Uuid, ManagedProcess> processMap = new HashMap<>();
```

Comment:
Uuid is the whole class rather than just the value. Better to use String and pass in the key as uuid.getValue() to return the real uuid.

Reply:
gonna leave it for now